src/main/java/org/apache/geronimo/microprofile/impl/jwtauth/jwt/KidMapper.java

```diff
@@ -79,9 +101,48 @@ private void init() {
                                     .collect(Collectors.toSet()))
                                 .orElseGet(HashSet::new);
         ofNullable(config.read("issuer.default", config.read(Names.ISSUER, null))).ifPresent(defaultIssuers::add);
+        jwksUrl = config.read("mp.jwt.verify.publickey.location", null);
+        readerFactory = Json.createReaderFactory(emptyMap());
+        ofNullable(jwksUrl).ifPresent(url -> {
+            HttpClient.Builder builder = HttpClient.newBuilder();
```

Comment:
guess minimum is a `protected void customize(HttpClient.Builder)` method - to be transparent I can't replace by custom code fillung the key map with that cause of missing sslContext+sslParameters setup, not asking to include it by default but enable it - guess the protected method is the fastest but I'm fine with an event too.